Tests/AppTests/PackageCollectionControllerTests.swift

```diff
@@ -58,10 +58,11 @@ class PackageCollectionControllerTests: AppTestCase {
                              summary: "summary 1").create(on: app.db)
 
         // MUT
+        let encoder = self.encoder
         try await app.test(
             .GET,
             "foo/collection.json",
-            afterResponse: { res async throws in
+            afterResponse: { @MainActor res async throws in
                 // validation
                 XCTAssertEqual(res.status, .ok)
                 let json = try res.content.decode(PackageCollection.self)
```

Comment:
The snapshot test here (and the others below) still fail due to the ordering issue. We _could_ skip them for `compiler(<6)` but it's safer to just let it be for now. If it becomes a problem in day-to-day running we can revisit that.